.github/workflows/fuzz-test.yml

```diff
@@ -0,0 +1,73 @@
+name: Runtime Fuzz Testing
+
+on:
+  # For the purpose of testing the workflow
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      num_of_iterations:
+        description: Number of iterations per fuzzable variable
+        required: False
+        default: 5
+
+jobs:
+  build:
+    runs-on:
+      - self-hosted
```

Comment:
Not really, I copied the format from the coverity workflow. However I have configured the workflow to work on the self-hosted machine, is there any reason why I can't use the self-hosted machine?

Reply:
> Ideally we want to get rid of / minimize all self-hosted machine to reduce maintainer cost. Currently only Coverity requires self-hosted machine due to it's not available (license) in the github machines.

I tried using the github hosted machine but [it failed with ssl certificate error](https://github.com/intel/fpga-runtime-for-opencl/actions/runs/5592330451/jobs/10224649829#step:7:13) :(

Reply:
@tylerzhao7684, I found this github [page ](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python)about how to run python install. Your code looked fine. I think the only difference might be the image you are using is a customized image that we built for the runtime [here](https://github.com/intel/fpga-runtime-for-opencl/tree/main/container).

We also have a workflow to update all the container [here](https://github.com/intel/fpga-runtime-for-opencl/blob/main/.github/workflows/container.yml). Be careful with this `container.yml` as this will update all the containers with the possibility of polluting them. You can read the comment inside to see how to just update one container.

I would suggest you to try the code in that github tutorial to see if you can get python install to run properly using the github runner + github default image. If you can get that to run, maybe something is missing in our container. Before you start to update our container, you can create a new container for your testing (so you don't have the chance to break other containers). I suspect might be some SSL module needs to be installed.

However, this is not the top priority for this task. We can merge this in as the self-hosted machine and fix it before the CC.